File: redome-front/src/app/doador/consulta/workup/agendamento.workup.dto.ts
import {BaseEntidade} from '../../../shared/base.entidade';

export class AgendamentoWorkupDTO extends BaseEntidade {
	public jsonToEntity(res: any) {
		throw new Error("Method not implemented.");
	}

	private _dataPrevistaDisponibilidadeDoador: Date;
	private _dataPrevistaLiberacaoDoador: Date;
	private _dataColeta: Date;
	private _dataInicioResultado: Date;
	private _dataFinalResultado: Date;
	private _dataLimiteWorkup: Date;

	private _idCentroColeta:number;
	private _necessitaLogistica:boolean;

	private _idFonteCelula: number;


	/**
	 * @returns Date
	 */
	public get dataFinalResultado(): Date {
		return this._dataFinalResultado;
	}
	/**
	 * @param  {Date} value
	 */
	public set dataFinalResultado(value: Date) {
		this._dataFinalResultado = value;
	}
	/**
	 * @returns number
	 */
	public get idCentroColeta(): number {
		return this._idCentroColeta;
	}
	/**
	 * @param  {number} value
	 */
	public set idCentroColeta(value: number) {
		this._idCentroColeta = value;
	}

	public get dataPrevistaDisponibilidadeDoador(): Date {
		return this._dataPrevistaDisponibilidadeDoador;
	}

	public set dataPrevistaDisponibilidadeDoador(value: Date) {
		this._dataPrevistaDisponibilidadeDoador = value;
	}

	public get dataColeta(): Date {
		return this._dataColeta;
	}

	public set dataColeta(value: Date) {
		this._dataColeta = value;
	}

	public get dataInicioResultado(): Date {
		return this._dataInicioResultado;
	}

	public set dataInicioResultado(value: Date) {
		this._dataInicioResultado = value;
	}

	public get dataLimiteWorkup(): Date {
		return this._dataLimiteWorkup;
	}

	public set dataLimiteWorkup(value: Date) {
		this._dataLimiteWorkup = value;
	}

	public get necessitaLogistica(): boolean {
		return this._necessitaLogistica;
	}

	public set necessitaLogistica(value: boolean) {
		this._necessitaLogistica = value;
	}

	public get dataPrevistaLiberacaoDoador(): Date {
		return this._dataPrevistaLiberacaoDoador;
	}

	public set dataPrevistaLiberacaoDoador(value: Date) {
		this._dataPrevistaLiberacaoDoador = value;
	}

    /**
     * Getter idFonteCelula
     * @return {number}
     */
	public get idFonteCelula(): number {
		return this._idFonteCelula;
	}

    /**
     * Setter idFonteCelula
     * @param {number} value
     */
	public set idFonteCelula(value: number) {
		this._idFonteCelula = value;
	}



}

File: redome/src/main/java/br/org/cancer/modred/controller/dto/AgendamentoWorkupDTO.java
package br.org.cancer.modred.controller.dto;

import java.io.Serializable;
import java.time.LocalDate;

/**
 * Classe referente aos parametros necessários para a finalização do agendamento de workup.
 * 
 * @author Fillipe Queiroz
 *
 */
public class AgendamentoWorkupDTO implements Serializable{
	private static final long serialVersionUID = -9184497700040395558L;

	private LocalDate dataPrevistaDisponibilidadeDoador;
	private LocalDate dataPrevistaLiberacaoDoador;
	private LocalDate dataColeta;
	private LocalDate dataInicioResultado;
	private LocalDate dataFinalResultado;
	
	private LocalDate dataLimiteWorkup;
	
	private Long idCentroColeta;
	private Boolean necessitaLogistica;
	private Long idFonteCelula;

	
	/**
	 * @return the dataPrevistaDisponibilidadeDoador
	 */
	public LocalDate getDataPrevistaDisponibilidadeDoador() {
		return dataPrevistaDisponibilidadeDoador;
	}
	
	/**
	 * @param dataPrevistaDisponibilidadeDoador the dataPrevistaDisponibilidadeDoador to set
	 */
	public void setDataPrevistaDisponibilidadeDoador(LocalDate dataPrevistaDisponibilidadeDoador) {
		this.dataPrevistaDisponibilidadeDoador = dataPrevistaDisponibilidadeDoador;
	}
	
	/**
	 * @return the dataPrevistaLiberacaoDoador
	 */
	public LocalDate getDataPrevistaLiberacaoDoador() {
		return dataPrevistaLiberacaoDoador;
	}
	
	/**
	 * @param dataPrevistaLiberacaoDoador the dataPrevistaLiberacaoDoador to set
	 */
	public void setDataPrevistaLiberacaoDoador(LocalDate dataPrevistaLiberacaoDoador) {
		this.dataPrevistaLiberacaoDoador = dataPrevistaLiberacaoDoador;
	}
	public LocalDate getDataInicioResultado() {
		return dataInicioResultado;
	}
	public void setDataInicioResultado(LocalDate dataInicioResultado) {
		this.dataInicioResultado = dataInicioResultado;
	}
	public LocalDate getDataFinalResultado() {
		return dataFinalResultado;
	}
	public void setDataFinalResultado(LocalDate dataFinalResultado) {
		this.dataFinalResultado = dataFinalResultado;
	}
	public Long getIdCentroColeta() {
		return idCentroColeta;
	}
	public void setIdCentroColeta(Long idCentroColeta) {
		this.idCentroColeta = idCentroColeta;
	}
	public Boolean getNecessitaLogistica() {
		return necessitaLogistica;
	}
	public void setNecessitaLogistica(Boolean necessitaLogistica) {
		this.necessitaLogistica = necessitaLogistica;
	}
	public LocalDate getDataLimiteWorkup() {
		return dataLimiteWorkup;
	}
	public void setDataLimiteWorkup(LocalDate dataLimiteWorkup) {
		this.dataLimiteWorkup = dataLimiteWorkup;
	}

	
	/**
	 * @return the dataColeta
	 */
	public LocalDate getDataColeta() {
		return dataColeta;
	}

	
	/**
	 * @param dataColeta the dataColeta to set
	 */
	public void setDataColeta(LocalDate dataColeta) {
		this.dataColeta = dataColeta;
	}

	
	/**
	 * @return the idFonteCelula
	 */
	public Long getIdFonteCelula() {
		return idFonteCelula;
	}

	
	/**
	 * @param idFonteCelula the idFonteCelula to set
	 */
	public void setIdFonteCelula(Long idFonteCelula) {
		this.idFonteCelula = idFonteCelula;
	}
	
	
	
}
